.gitignore

```diff
@@ -138,3 +138,4 @@ mlruns/
 *.ckpt
 pytorch\ lightning
 test-reports/
+wandb
```

Comment:
how it happens that this folder is created?

Reply:
wandb logger always makes this folder, we can't avoid this. I have it too. I think it's ok to add it to gitignore.